controllers/articles.controllers.js

```diff
@@ -12,8 +13,17 @@ exports.getArticleById = (req, res, next) => {
     const { article_id } = req.params;
     fetchArticleById(article_id)
         .then((article) => {
-         res.status(200).send({ article })
+            res.status(200).send({ article })
         })
         .catch((err) => next(err))
+}
 
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+    Promise.all([checkArticleExists(article_id), fetchCommentsByArticleId(article_id)])
```

Comment:
Can just make use of fetchArticleById instead of making a new model

Reply:
Oh of course!!! I have refactored this, thanks so much for this feedback!